Sources/WordPressUI/Extensions/Gravatar/GravatarURL+Util.swift

```diff
@@ -13,12 +12,15 @@ extension GravatarURL {
     public static func url(for email: String,
                            preferredSize: ImageSize? = nil,
                            gravatarRating: Rating? = nil,
-                           defaultAvatarOption: DefaultAvatarOption? = .status404) -> URL? {
```

Comment:
Is it possible there was a merge error here? 

Because "defaultAvatarOption: DefaultAvatarOption? = .status404" belongs to the older state of the SDK.